Code/Framework/AzFramework/AzFramework/Network/AssetProcessorConnection.cpp

```diff
@@ -11,6 +11,7 @@
 #include <AzCore/Math/Crc.h>
 #include <AzCore/Platform.h>
 #include <AzCore/Casting/numeric_cast.h>
+#include <AzCore/std/algorithm.h>
```

Comment:
is this header needed?

Reply:
Yes it is needed, because the AssetProcessorConnection.cpp is using the [AZ_TRAIT_THREAD_AFFINITY_MASK_ASSET_PROCESSOR_CONNECTION_THREAD](https://github.com/o3de/o3de/blob/e194ce2d18a770e34c4fbe8208dc0438dbb7cd92/Code/Framework/AzCore/Platform/Linux/AzCore/AzCore_Traits_Linux.h#L97) macro which now uses the `AZStd::clamp` function which requires algorithm.h for the symbol